src/Components/ArticleComponents/CommentContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import CommentCard from "./CommentCard";
+import { useState, useEffect } from "react";
+import { getCommentsByArticleById } from "../../api";
+
+export default function CommentsContainer({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [commentsLoading, setCommentsLoading] = useState(true);
+
+  useEffect(() => {
+    setCommentsLoading(true);
+    getCommentsByArticleById(article_id).then((res) => {
+      res.sort((b, a) => a.votes - b.votes);
```

Comment:
why are you sorting by votes here? do you have the ability to query this on your be?